style/ruby/.rubocop.yml

```diff
@@ -90,6 +90,13 @@ Metrics/AbcSize:
                  branches, and conditions.
   Enabled: false
 
+Metrics/BlockLength:
+  CountComments: true  # count full line comments?
+  Max: 25
+  ExcludedMethods: []
```

Comment:
This is the default in RuboCop. Are you thinking that it might be nice to explicit here?